stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1234,6 +1249,11 @@ public final class TestSuite {
     case .parameterized(code: let code, _):
       code(parameter!)
     }
+
+#if SWIFT_RUNTIME_ENABLE_LEAK_CHECKER
```

Comment:
Yes.  I don't think we want the leak checker enabled for the entire test suite just yet is the thing.  It will significantly slow down the test suite.